.dict_custom.txt

```diff
@@ -99,3 +100,7 @@ unhandled
 unallocated
 mypy
 destructor
+destructor's
+VALGRIND
+initialize
+initialized
```

Comment:
These words probably exist in the dictionary already but we have a UK dictionary not a US dictionary so they will be spelled:
- initialise
- initialised

Reply:
Fixed https://github.com/pyccel/pyccel/pull/1580/commits/a603c43dc6faf2844942ca4541fab1d52b177432.